apps/discordbot/bot.py

```diff
@@ -37,16 +42,33 @@ async def on_ready(self):
             self.test_channel = get(self.test_guild.channels, id=TEST_CHANNEL_ID)
             self.hoser_role = get(self.csua_guild.roles, id=HOSER_ROLE_ID)
 
+    async def verify_member_email(self, user): 
+        channel = user.dm_channel
+        def check_msg(msg, user):
+            return msg.channel == channel
+        got_email = False
+        while not got_email:
+            msg = await self.wait_for('message', check=check_msg)
+            try:
+                validate_email(msg)
```

Comment:
This doesn't check that their email is a berkeley.edu email. Is that intentional?

Reply:
added berkeley.edu email check. At the moment, it has to be @ berkeley.edu so prevent people from doing shenanigans with lists or somesuch.